src/components/screen-header/index.tsx

```diff
@@ -8,6 +8,8 @@ import React, {useMemo, useState} from 'react';
 import {StyleSheet, useTheme} from '@atb/theme';
 import HeaderButton, {ButtonModes, HeaderButtonProps} from './HeaderButton';
 import ThemeText from '@atb/components/text';
+import {AlertContext} from '@atb/alerts/AlertsContext';
+import AlertBox from '@atb/components/alert-box';
```

Comment:
Should alert-box be part of alerts instead?

Reply:
Moved